cli_tests.py

```diff
@@ -29,38 +29,38 @@ def test_table_subcommand_with_no_additional_arguments(self):
 
     def test_table_generation_for_width_8_and_poly_0x1D(self):
         expected_output = io.StringIO(inspect.cleandoc("""
```

Comment:
I have checked the table, looks like the one you added is correct. Apparently this test was setup incorrectly.